### PR TITLE
Negative tests

### PR DESCRIPTION
**Pull request by siposb94**
Added negative tests

JerryScript-DCO-1.0-Signed-off-by: Balint Sipos siposb@inf.u-szeged.hu

### CODE REVIEW

**Comment by LaszloLango** [comment]
typo in the filename: `tests/jerry/arguments-pharse.js` -> `tests/jerry/arguments-parse.js`

File: tests/jerry/arguments-pharse.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+}
+
+check_parse_error("f_args(1 2 3)");
+check_parse_error("f_args(1;2;3)");
+check_parse_error("f_args(())");
+check_parse_error("f_args(1,2,3");
+check_parse_error("f_args 1,2,3)");
+check_parse_error("f_args 1,2,3");
+check_parse_error("f_args 1;2;3");
+check_parse_error("f_args{1,2,3}");
```
Comment (robertsipka) [inline tests/jerry/arguments-pharse.js]: Please add a new line and use two spaces for indents

File: tests/jerry/arithmetic-parse.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+parse("c = a -/ b");
+parse("c = a +% b");
+parse("c = a -% b");
+
+parse("a =* b");
+parse("a =/ b");
+parse("a =% b");
+
+parse("c = a+");
+parse("c = a-");
```
Comment (robertsipka) [inline tests/jerry/arithmetic-parse.js]: ditto

File: tests/jerry/for-in-parse.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    "   obj[prop] += 4;"
+parse(forIn)
+
+var forIn =
+    "foreach (var prop in obj)" +
+    "   obj[prop] += 4;"
+parse(forIn)
+
+
+
```
Comment (robertsipka) [inline tests/jerry/for-in-parse.js]: Too much new line here.

File: tests/jerry/for_parser.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+var doubleFor =
+"for(i=0;i<5;i++) { "+ 
+"   for(j=0;j<5;j++) { print('i: '+i+' j: '+j) }";
+parse(doubleFor)
+
+var doubleFor2 =
+"for(i=0;i<5;i++) { "+ 
+"   for(j=0;j<5;j++) { print('i: '+i+' j: '+j)";
+parse(doubleFor2)
```
Comment (robertsipka) [inline tests/jerry/for_parser.js]: ditto

File: tests/jerry/if_parser.js [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+"if (false) print('if statement') " +
+"elif (false) print('else if statement') " +
+"else print('else statement') "
+test_parse_error(elseif2)
+
+var elseif3 =
+"if (false) print('if statement') " +
+"else (false) print('else if statement') " +
+"else print('else statement') "
+test_parse_error(elseif3)
```
Comment (robertsipka) [inline tests/jerry/if_parser.js]: ditto

**Pull request merged**